template.txt

```diff
@@ -0,0 +1,63 @@
+A template for making this project
+
+
+answer = choice(list_of_fruits)
+life = 5
+chosen_letters = []
+game_over = False
+
+#Task 2, 3, 4, 5
+def display_blanks():
+  """
+  This function will be called over and over again to display the correctly-guessed letters. Letters not guessed correctly will be displayed with an underscore symbol.
+  """
+  #Task 3 - define empty string for display_txt
+  display_txt = ""
+  
+  #Task 4 - loop through each letter of the answer and add the correct letters or underscore
+  for letter in answer:
+    if letter in chosen_letters:
+      display_txt += letter + " "
+    else:
+      display_txt += "_ "
+  
+  #Task 5 - return the complete display_txt
+  return display_txt + "\n"
+  
+#print the blank spaces based on the length of
+#the answer when the game starts  
+print("_ " * len(answer))
```

Comment:
It better if the word is separated from each other like Christmastable just add the space or make it a new word such as remove the table part

Reply:
okay